graf2d/gpad/inc/TRatioPlot.h

```diff
@@ -0,0 +1,219 @@
+// @(#)root/hist:$Id$
+// Author: Rene Brun   10/12/2001
+
+/*************************************************************************
+ * Copyright (C) 1995-2001, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_TRatioPlot
+#define ROOT_TRatioPlot
+
+
+//////////////////////////////////////////////////////////////////////////
+//                                                                      //
+// ROOT_TRatioPlot                                                      //
+//                                                                      //
+// A collection of histograms                                           //
+//                                                                      //
+//////////////////////////////////////////////////////////////////////////
+
+#include "TH1.h"
+#include "TPad.h"
+#include "TGraphAsymmErrors.h"
+#include "TGraphErrors.h"
+#include "TGraph.h"
+#include "TAxis.h"
+#include "TGaxis.h"
+#include "TH1F.h"
+#include "TFitResultPtr.h"
+#include "THStack.h"
```

Comment:
You don't need most (any?) of these `#include`s; you can replace them with a forward declaration.

Where you need the `#include` it should be surrounded with the include guard, see other ROOT headers.